scene/gui/tab_container.cpp

```diff
@@ -293,6 +293,8 @@ void TabContainer::_notification(int p_what) {
 }
 
 int TabContainer::_get_tab_width(int p_index) const {
+	if (_get_tabs().empty())
```

Comment:
That's only hiding a bug, no?

There should be a `ERR_FAIL_COND_V(p_index >= _get_tabs().size(), 0)` or similar, and the situation where `_get_tab_width` is called on an empty tabs should be prevented so that the error is not triggered.

Reply:
ok, fixed